src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -191,9 +191,12 @@ public static int MainInternal(string[] args, CommandOutputLogger log)
                     // Log the stack trace as verbose output.
                     log.LogVerbose(e.ToString());
 
-                    exitCode = 1;
+                    if (e.GetType().Name == "CommandParsingException")
```

Comment:
```suggestion
                    if (e.GetType() == typeof(CommandParsingException))
```

Reply:
I just realized that my suggestion isn't even the best suggestion

```suggestion
                    if (e is CommandParsingException)
```